.circleci/build-workbase-win/build-workbase-win.py

```diff
@@ -0,0 +1,166 @@
+])
+
+print('Generating password for instance')
+instance_password = subprocess.check_output([
+    'openssl', 'rand', '-base64', '12'
+]).strip()
+
+instance_name = 'circleci-{}-{}'.format(
+    os.getenv('CIRCLE_JOB'), os.getenv('CIRCLE_BUILD_NUM'))
+
+print('Generating bootup script for instance [{}]'.format(instance_name))
+with tempfile.NamedTemporaryFile(suffix='.ps1') as powershell_script:
+    with open('.circleci/build-workbase-win/instance-setup-template.ps1') as template:
+        powershell_script.write(template.read().replace('INSTANCE_PASSWORD', instance_password).encode())
+        powershell_script.flush()
+
+    print('Provisioning instance [{}]'.format(instance_name))
+    subprocess.check_call([
+        'gcloud', 'compute', 'instances', 'create', instance_name,
+        '--image-project', 'windows-cloud', '--image-family', 'windows-2019',
+        '--machine-type', 'n1-standard-1', '--metadata-from-file',
+        'sysprep-specialize-script-ps1={}'.format(powershell_script.name)
+    ])
+
+print('Storing instance\'s external IP')
+instance_ip = subprocess.check_output([
+    'gcloud', '--format', 'value(networkInterfaces[0].accessConfigs[0].natIP)',
+    'compute', 'instances', 'list', '--filter', 'name={}'.format(instance_name)
+])
+
+
+try:
+    print('Waiting for instance to some alive (sshd)')
+    wait_for_ssh(instance_ip, 'circleci', instance_password)
+
+    print('Executing command remotely')
+    ssh('dir', instance_ip, 'circleci', instance_password, check_call=True)
+
+    print('Executing PowerShell command remotely [Get-LocalUser]')
+    ssh('powershell Get-LocalUser', instance_ip, 'circleci', instance_password, check_call=True)
+
+    print('Installing git')
+    ssh('choco install git -y', instance_ip, 'circleci', instance_password, check_call=True)
+
+    print('Installing nodejs 8')
+    ssh('choco install nodejs -y --version 8.15.0', instance_ip, 'circleci', instance_password, check_call=True)
+
+    print('Cloning workbase')
+    ssh(
+        'refreshenv && git clone {repo_url} repo && cd repo && git checkout {repo_commit}'.format(
+            repo_url=replace_git_url_to_https(os.getenv('CIRCLE_REPOSITORY_URL')),
+            repo_commit=os.getenv('CIRCLE_SHA1')),
+        instance_ip, 'circleci', instance_password, check_call=True)
+
+    print('[Remote]: npm install')
+    ssh('refreshenv && cd repo && npm install', instance_ip, 'circleci', instance_password, check_call=True)
+
+    print('[Remote]: npm run build')
+    ssh('refreshenv && cd repo && npm run build', instance_ip, 'circleci', instance_password, check_call=True)
+
+    print('Copying built Workbase executable from remote to local')
+    scp('C:\\Users\\circleci\\repo\\build\\GRAKNW~1.EXE', './GRAKN.exe', instance_ip, 'circleci', instance_password, check_call=True)
+
+    print('Verifying local file')
+    subprocess.check_call(['file', './GRAKN.exe'])
+
+
+finally:
+    pass
+    # print('Remove instance')
```

Comment:
Why is the cleanup code commented out? Where are we doing the cleanup?

Reply:
@lolski cleanup restored in https://github.com/graknlabs/workbase/pull/7/commits/457305946b1641f6c7c27219522017b381db1ac9